src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ProjectReference.xaml

```diff
@@ -41,6 +53,55 @@
         </BoolProperty.DataSource>
     </BoolProperty>
 
+    <StringProperty Name="Culture" 
+                    ReadOnly="True"
+                    DisplayName="Culture" 
+                    Description="The value of the culture field from the assembly metadata.">
+    </StringProperty>
+
+    <StringProperty Name="Description" 
+                    ReadOnly="True" 
+                    DisplayName="Description" 
+                    Description="The value of the Title field from the assembly metadata.">
+    </StringProperty>
+
+    <BoolProperty Name="EmbedInteropTypes"
+                  DisplayName="Embed Interop Types"
+                  Description="Indicates whether types defined in this assembly will be embedded into the target assembly.">
+        <BoolProperty.DataSource>
```

Comment:
@davkean Isn't the property name in the project file `Private` instead of `CopyLocal`?